tests/plugins/wysiwygarea/manual/detachobserver.html

```diff
@@ -0,0 +1,33 @@
+<div id="editor">
+	<p>Lorem ipsum dolor sit amet.</p>
+</div>
+<button id="triggerButton" disabled>Trigger test</button>
+
+<script>
+	( function() {
+		if ( bender.tools.env.mobile || ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) ) {
+			return bender.ignore();
+		}
+
+		var triggerButton = document.getElementById( 'triggerButton' );
+		triggerButton.addEventListener( 'click', function() {
+			delete CKEDITOR;
+
+			var element = document.createElement( 'div' );
+			document.body.appendChild( element );
+		} );
+
+		CKEDITOR.replace( 'editor', {
+			on: {
+				instanceReady: function ( evt ) {
+					evt.editor.destroy();
+				},
```

Comment:
WDYT about moving destroy logic into button's click action? It seems counterintuitive to me that a manual test does something without even my knowledge.

Additionally, will the error connected to the removed `CKEDITOR` namespace even appear if the observer should be disconnected right after the editor's destroy? Probably these two tests should be separate.

Reply:
I thought about it previously. However, the `detroy` is not tested here. The significant action takes place on the button click action (force changed to trigger MutationObserver). So - still you decide when exactly the critical logic will be executed.

I'll separate those two cases.